src/Adapters/Cache/InMemoryCache.js

```diff
@@ -0,0 +1,69 @@
+const DEFAULT_CACHE_TTL = 60 * 1000;
+
+
+export class InMemoryCache {
+  constructor({
+    ttl = DEFAULT_CACHE_TTL
+  }) {
+    this.ttl = ttl;
+    this.cache = Object.create(null);
+  }
+
+  get(key) {
+    let record = this.cache[key];
+    if (record == null) {
+      return null;
+    }
+
+    // Has Record and isnt expired
+    if (isNaN(record.expire) || record.expire >= Date.now()) {
+      return record.value;
```

Comment:
``InMemoryCache`` isn't actually a ``CacheAdapter``,

there was a cache already in the application which needed to be synchronous (``AppCache``), I moved that implementation here and added a TTL option to it, so we don't have two cache classes.

the ``InMemoryCacheAdapter`` wraps this in promises.